doc/index.rst

```diff
@@ -82,6 +81,32 @@ Last but not least, daal4py allows :ref:`getting input data from streams <stream
         algo.compute(input)
     result = algo.finalize()
 
+oneAPI and GPU support in daal4py
+---------------------------------
+daal4py provide support of oneAPI conepts such as context and queues. This allow execution of
+algorithms on different devices, particularly GPUs. This implemented via 'with sycl_context("xpu")'
+blocks that will direct execution to selected device type. gpu, cpu and host devices are supported
+at the moment. Same aproach implemented for scikit-learn patching, so scikit-learn programs can be
+executed on GPU devices as well.   
+
+Intel CPU/GPU optimizations patching:
+```py
+from daal4py.sklearn import patch_sklearn
+from daal4py.oneapi import sycl_context
+patch_sklearn()
+
+from sklearn.svm import SVC
+from sklearn.datasets import load_digits
+digits = load_digits()
+X, y = digits.data, digits.target
+with sycl_context("gpu"):
+    clf = SVC().fit(X, y)
+    res = clf.predict(X)
+```
+For GPU execution DPC++ runtime(compiler runtime and driver) are requered - DPC++ system
```

Comment:
It should be said that the runtime can be obtained through a conda or pip channel.

Reply:
done